configs/ratelimit.toml

```diff
@@ -0,0 +1,89 @@
+# An example configuration for benchmarking Pelikan Segcache with an 80/20
+# read/write mix in a keyspace of 1M keys using 32B keys and 128B values.
+
+[general]
+# specify the protocol to be used
+protocol = "memcache"
+# the interval for stats integration and reporting
+interval = 5
+# the number of intervals to run the test for
+duration = 150
+# optionally, we can write some detailed stats to a file during the run
+#json_output = "stats.json"
+# run the admin thread with a HTTP listener at the address provided, this allows
+# stats exposition via HTTP
+admin = "127.0.0.1:9090"
+# optionally, set an initial seed for the PRNGs used to generate the workload.
+# The default is to intialize from the OS entropy pool.
+#initial_seed = "0"
+
+[debug]
+# choose from: error, warn, info, debug, trace
+log_level = "info"
+# optionally, log to the file below instead of standard out
+# log_file = "rpc-perf.log"
+# backup file name for use with log rotation
+log_backup = "rpc-perf.log.old"
+# trigger log rotation when the file grows beyond this size (in bytes). Set this
+# option to '0' to disable log rotation.
+log_max_size = 1073741824
+
+[target]
+# specify one or more endpoints as IP:PORT pairs
+endpoints = [
+	"127.0.0.1:12321",
+]
+
+[client]
+# number of threads used to drive client requests
+threads = 4
+# the total number of connections to each endpoint
+poolsize = 20
+# the connect timeout in milliseconds
+connect_timeout = 10000
+# set the timeout in milliseconds
+request_timeout = 1000
+
+[workload]
+# the number of threads that will be used to generate the workload
+threads = 1
+
+[workload.ratelimit]
+# set a global ratelimit for the workload
+ratelimit = 10_000
+ratelimit_end = 100_000
+step_size = 10_000
+step_interval = 15
```

Comment:
15 what? I think we should use `humantime` here so it can be seconds/minutes/hours

Reply:
It is in seconds already. Will add a comment for that when cleaning up.